### ISSUE TITLE
filename incorrect

### ISSUE THREAD

**Issue by klauswyser** [goord/ece2cmor3#30]
The filename (after cmorisation) should follow the template on p13 of https://docs.google.com/document/d/1h0r8RZr_f3-8egBMMh7aqLwy3snpD6_MrDz1q8n5XUk/edit
```
file name = <variable_id>_<table_id>_<source_id>_<experiment_id >_<member_id>_<grid_label>[_<time_range>].nc
```
Currently, the order of the filename components is slightly different, i.e. <source_id> and <experiment_id > have switched position.

**Comment by goord** [comment]
Klaus, is the default template that CMOR uses correct?

**Comment by goord** [comment]
I believe Jon Seddon has sent an email about this, and it has been fixed in a recent version of CMOR3.

**Comment by klauswyser** [comment]
With CMOR 3.2.3 I get the correct filenames after removing 
` "output_file_template":         "<variable_id><table><experiment_id><source_id><variant_label><grid_label>", `
from the configuration file.

**Comment by klauswyser** [comment]
Solved with CMOR 3.2.3
